Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+pipeline {
+     agent {
+         docker {
+             image 'maven:3-alpine'
+         }
+     }
+     stages {
+         stage('Build') {
+             steps {
+                 sh 'mvn -B -DskipTests clean install'
```

Comment:
Can you just drop a link to why we need `-B` here, for documentation?

Reply:
The -B is a batch-mode. The batch-mode will automatically use default values instead of asking you via prompt for those values, which we don't want to happen in a Jenkins build container. The batch-mode can also be activated via --batch-mode or -B on the command line.